### PR TITLE
Replace tableconvertor with APC

### PR DESCRIPTION
**Pull request by adamwalach**
**Related issues**
* https://github.com/kyma-project/kyma/issues/2790

### CODE REVIEW

File: pkg/controller/controller_instance.go [hunk 1]
```diff
@@ -539,6 +539,20 @@ func (c *controller) reconcileServiceInstanceAdd(instance *v1beta1.ServiceInstan
 		return nil
 	}
 
+	if c.resolveServiceInstanceUserSpecifiedClassAndPlan(instance) {
+		updatedInstance, err := c.updateServiceInstanceStatus(instance)
+		if err != nil {
+			// There has been an update to the instance. Start reconciliation
```
Comment (mszostok) [inline pkg/controller/controller_instance.go]: this comment does not look like being a proper one

**Review by mszostok — COMMENTED** [review]
so far it looks good :)

File: pkg/controller/controller_servicebroker.go [hunk 2]
```diff
@@ -658,6 +658,7 @@ func (c *controller) updateServiceBrokerCondition(broker *v1beta1.ServiceBroker,
 
 	pcb := pretty.NewServiceBrokerContextBuilder(toUpdate)
 	updateCommonStatusCondition(pcb, toUpdate.ObjectMeta, &toUpdate.Status.CommonServiceBrokerStatus, conditionType, status, reason, message)
+	toUpdate.Status.LastConditionState = getServiceBrokerLastConditionState(toUpdate.Status.CommonServiceBrokerStatus)
```
Comment (piotrmiskiewicz) [inline pkg/controller/controller_servicebroker.go]: is it tester somewhere? I've just commented the line and all unit tests are green

Comment (piotrmiskiewicz) [reply pkg/controller/controller_servicebroker.go]: why do not put the update  LastConditionState to updateCommonStatusCondition function?

**Review by piotrmiskiewicz — CHANGES_REQUESTED** [review]

**Review by piotrmiskiewicz — APPROVED** [review]

File: pkg/controller/controller_servicebroker.go (hunk 2 above)
Comment (adamwalach) [reply pkg/controller/controller_servicebroker.go]: I added tests, and move the update to funtion

**Pull request merged**